source/about/accessibility.md

```diff
@@ -1,44 +1,26 @@
 # Accessibility at arXiv
 
-arXiv is committed to creating an accessible experience for all users. Below is our thinking behind this commitment and our roadmap for reaching and sustainably maintaining an accessible platform for the entire arXiv community.
+Accessibility and access are not the same. Accessibility *ensures access regardless of disability*. arXiv is committed to making our platform and the research we host as accessible as we can. Because if it is not accessible, we can't call it open science.
```

Comment:
This sentence seems like an argument against an unstated strawman. Can we more directly state why making articles accessible to those with impairment is a good thing -- but not get too hung up on the point?

(You could always start with "We hold these truths to be self-evident" ...)